pywt/_multidim.py

```diff
@@ -237,7 +155,11 @@ def dwtn(data, wavelet, mode='symmetric'):
         raise ValueError("Input data must be at least 1D")
     coeffs = [('', data)]
 
-    for axis in range(data.ndim):
+    if axes is None:
+        axes = range(data.ndim)
+    axes = (a + data.ndim if a < 0 else a for a in axes)
+
+    for axis in sorted(axes):
```

Comment:
I think the use of sorted axes needs documenting. In `np.fft.fftn` the axes are transformed over in the order given.